GameServerLib/API/ApiFunctionManager.cs

```diff
@@ -357,6 +392,40 @@ public static List<IAttackableUnit> GetUnitsInRange(Vector2 targetPos, float ran
             return _game.ObjectManager.GetUnitsInRange(targetPos, range, isAlive);
         }
 
+        /// <summary>
+        /// Acquires the closest alive or dead AttackableUnit within the specified range of a target position.
+        /// </summary>
+        /// <param name="targetPos">Origin of the range to check.</param>
+        /// <param name="range">Range to check from the target position.</param>
+        /// <param name="isAlive">Whether or not to return alive AttackableUnits.</param>
+        /// <returns>Closest AttackableUnit.</returns>
+        public static IAttackableUnit GetClosestUnitInRange(Vector2 targetPos, float range, bool isAlive)
+        {
+            var units = _game.ObjectManager.GetUnitsInRange(targetPos, range, isAlive);
+            var orderedUnits = units.OrderBy(unit => Vector2.DistanceSquared(targetPos, unit.Position));
+
+            return orderedUnits.First();
+        }
+
+        /// <summary>
+        /// Acquires the closest alive or dead AttackableUnit within the specified range of another unit.
+        /// </summary>
+        /// <param name="target">Origin of the range to check.</param>
+        /// <param name="range">Range to check from the target position.</param>
+        /// <param name="isAlive">Whether or not to return alive AttackableUnits.</param>
+        /// <returns>Closest AttackableUnit.</returns>
+        public static IAttackableUnit GetClosestUnitInRange(IAttackableUnit target, float range, bool isAlive)
+        {
+            var units = _game.ObjectManager.GetUnitsInRange(target.Position, range, isAlive);
+            var orderedUnits = units.OrderBy(unit => Vector2.DistanceSquared(target.Position, unit.Position));
```

Comment:
Here you can call `GetClosestUnitInRange(Vector2 targetPos, float range, bool isAlive)`